entities/params/templates.go

```diff
@@ -16,8 +17,9 @@ func (p *PostTemplate) TrimSpaces() {
 
 // PutTemplate represents request body for PUT /api/templates
 type PutTemplate struct {
-	Content string `json:"content" form:"content" validate:"required,html"`
-	Subject string `json:"subject" form:"subject" validate:"required,max=191"`
+	HTMLPart string `form:"html_part" validate:"required,html"`
```

Comment:
@Dzalevski do all of the fields need to be required on put?

Reply:
Yes its the same as create, we send all fields.